### PR TITLE
Set the default mac ageing time to 600 seconds

### PR DESCRIPTION
**Pull request by zhenggen-xu**
The current mac ageing was disabled, this could lead the mac address
table to increase over time and lead to resource and performance issues.

Signed-off-by: Zhenggen Xu <zxu@linkedin.com>

**- What I did**
Set mac ageing time default to 300 seconds
 
**- How I did it**
set the ageing time in switch.json generated by switch.j2,  it will be loaded during the init or service restart.

**- How to verify it**
Before the fix, macs learnt were never aged out.
After the fix,  after ~5 minutes, the macs learnt but inactive were aged out. 

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by lguohan** [comment]
for this one, we need ansible test to validate this.

**Comment by jipanyang** [comment]
Just curious, why is 300 seconds aging time chosen as default?

**Comment by zhenggen-xu** [comment]
@lguohan We will add ansible test cases, but that is on a different repo. It also makes sense to have this merged before the other PR.

**Comment by zhenggen-xu** [comment]
> Just curious, why is 300 seconds aging time chosen as default?

It is de facto default value, at least for most big vendors.

**Comment by prsunny** [comment]
@zhenggen-xu , we have the arp_update interval also as 300sec. Just thinking of a case where mac ages out but neighbor entry still points to this MAC.

**Comment by zhenggen-xu** [comment]
@prsunny The arp update and HW MAC aging are some what agnostic.  On many platforms,  arp entry is doing the IP to MAC translation and have the neighbor destination information embedded into the entry and not relying on MAC table.  Even on platform that is using the MAC table info for neighbor destination, if MAC aged out,  traffic hit the neighbor entry should flood.  

Also, arp update was mostly dealing with VLAN facing ports where traffic hit HW not control plane and the entry could be aged-out from control plane (linux kernel). In those scenarios,  HW MAC should not be aged out as long as the traffic hit HW.  But anyway, even MAC aged out before ARP,  it shouldn't be a problem as mentioned above.

So overall, I don't see issues here.

**Comment by prsunny** [comment]
@zhenggen-xu , that was my point, in platforms where neighbor points to mac table, it starts flooding which will be different if there was no aging. I'm just saying this would be a behavioral change.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
@qiluo-msft , I do not think that the vs docker load the swich.json by default. But it seems the right thing to do. We should set the mac aging and then do a warm reboot test.

**Comment by zhenggen-xu** [comment]
> Will it break warm-reboot vs test?
> https://github.com/Azure/sonic-swss/blob/5984e3aeba59249eff71bd13666bb7b64bd493c8/tests/test_fdb_warm.py#L79

The warm-reboot vs test case does not assert after getting the FDB aging timer, so it won't break in any case.  Also, the aging timer is only applicable to HW, so it should not fail any test cases in vs environment,  vs get or not get the timer value should not matter.  For the real HW test, the process will disable aging during the warm reboot as we talked before, it will need restore it after come back, it should be done by swss.json, so I think it is covered there as well.

**Comment by lguohan** [comment]
retest this please

**Comment by qiluo-msft** [comment]
Why closed?

**Comment by zhenggen-xu** [comment]
it was closed by accident when I clean up my branches.

File: dockers/docker-orchagent/switch.json.j2 [hunk 1]
```diff
@@ -12,7 +12,8 @@
     {
         "SWITCH_TABLE:switch": {
             "ecmp_hash_seed": "{{ hash_seed }}",
-            "lag_hash_seed": "{{ hash_seed }}"
+            "lag_hash_seed": "{{ hash_seed }}",
+            "fdb_aging_time": "300"
```
Comment (lguohan) [inline dockers/docker-orchagent/switch.json.j2]: -> 600 as default.

Comment (zhenggen-xu) [reply dockers/docker-orchagent/switch.json.j2]: I am fine to change it to 600 seconds to be on the safer side comparing to ARP update interval.

**Comment by yxieca** [comment]
@lguohan  do you have more concern?

**Comment by yxieca** [comment]
retest this please

**Comment by qiluo-msft** [comment]
@lguohan Should we cherry-pick it into 201803 since it is a general feature missing?

**Pull request merged**